docs/api-docs/headless/carts.md

```diff
@@ -4,80 +4,112 @@
 
 In this section, we will explain how to use the Carts API to create and manage carts. Additionally, we will discuss how to redirect shoppers from a headless storefront to the BigCommerce hosted cart and checkout pages.
 
+<!-- theme: info -->
+> #### Note
+> The Carts API supports selling in different countries by allowing overrides for locale and product details. Merchants can supply a cart's locale, alternative product names, and option values.  
```

Comment:
```suggestion
> The Carts API supports selling in different markets by allowing locale-based overrides. To use alternative product names and option values, <specify those values on the product object and then add the corresponding locale to a cart>.  
```

@bc-traciporter I've added the carrots around approximate text that captures this gist. I leave the actual writing to you, as well the fact checking if this seems wrong or doesn't make sense. I am not 100% about whether option values are pre-existing on the product object, or defined ad hoc in the cart. If they're ad hoc, the gist could be something like this:

```suggestion
> The Carts API supports selling in different markets by allowing locale-based overrides. To use alternative product details, <use the product object to specify product info for a given locale, then add the corresponding locale to a cart. Your application can use the locale to set its own option values>.  
```

Reply:
never mind, this is wrong!